.vscodeignore

```diff
@@ -12,3 +12,7 @@ client/**
 server/**
 !server/target/*/release/slice-language-server
 !server/target/*/release/slice-language-server.exe
+
+# Slice
+slice/**/Internal
+slice/**/!(*.slice)
```

Comment:
I didn't know this syntax existed. Is it the same as doing `!ice/**/*.slice`?

Reply:
I would think so but I couldnt get `!slice/**/*.slice` to work